src/concrete/ml/sklearn/base.py

```diff
@@ -1277,7 +1277,7 @@ def __init__(self, n_bits: int):
         """
         self.n_bits: int = n_bits
 
-        #: The model's inference function. Is None if the model is not fitted.
+        # _tree_inference: The model's inference function. Is None if the model is not fitted.
```

Comment:
no, this is not how this docstring work ! you should revert this change

Reply:
@RomanBredehoft I think you should explain why you think that

Reply:
if I remember correctly, it's related to sphinx, they display class attributes like this, without the name

maybe we could check again the source of this docstring but one thing that is sure is that all other class attributes comments we have are starting by `#:` directly, that's why I think this change is a mistake

Reply:
It seems to me that you've already told me to put the name of the attribute before the comment.
That's why I thought it was an oversight.

No problem, I'll remove it in that case.